sale_discount_display_amount/hooks.py

```diff
@@ -2,41 +2,48 @@
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
 import logging
 
-from odoo import SUPERUSER_ID
-from odoo.api import Environment
 from odoo.tools.sql import column_exists, create_column
 
-_logger = logging.getLogger(__name__)
+logger = logging.getLogger(__name__)
```

Comment:
This notation is widely used to indicate that should not be used externally. You should let it as is.